language-design.md

```diff
@@ -74,7 +120,17 @@ implementation choices from users?
 
 **Response**
 
+Implementing an internal DSL will give it the feeling of the host language, allowing more
+familiarity with the syntax, but it won't give the developers as much control over what the
+users can do. 
```

Comment:
Why would the developers have less control? It seems like within the context of the DSL devs should still have absolute control over what the user can and can't do, and what the user does outside the context of the DSL shouldn't affect what the user can to within it.